code/_globalvars/lists/mobs.dm

```diff
@@ -28,6 +28,7 @@ GLOBAL_LIST_EMPTY(spidermobs)				//all sentient spider mobs
 GLOBAL_LIST_EMPTY(bots_list)
 GLOBAL_LIST_EMPTY(living_cameras)
 GLOBAL_LIST_EMPTY(aiEyes)
+GLOBAL_LIST_EMPTY(got_xmas_presents) // Seasonal!

```

Comment:
move this to the file that uses it.

Reply:
I don't think it should even be made until someone would be added to the list, it wouldn't be used for the majority of rounds.

Reply:
Yeah, I'll move it to a static list on the type.